bedrock/mozorg/helpers/misc.py

```diff
@@ -462,8 +462,10 @@ def absolute_url(url):
 
     if url.startswith('//'):
         prefix = 'https:'
-    else:
+    elif url.startswith('/'):
```

Comment:
This is only a tiny change and has a test below, so looks OK to me. But I'd appreciate it if either @pmclanahan or @jgmize could take a quick look and give their blessing.